cross_country_aggregate_dlt.py

```diff
@@ -505,7 +474,7 @@ def quality_boost_func_exact():
     # This doesn't check by year on purpose as new years may be added to pipeline
     # without the CCI excel being updated.
     boost_countries = dlt.read('quality_boost_country').select('country_name').distinct()
-    quality_cci_func = (spark.table(f'{catalog}.{boost_intermediate_schema}.quality_functional_gold')
+    quality_cci_func = (spark.table(f'{catalog}.{boost_staging_schema}.quality_functional_gold')
```

Comment:
Is this a temp change that is meant only for testing? If so, I wonder if we could just env var for the schema and default to the prod schema and overwrite it in the staging pipeline? That way it wouldn't require code changes for staging testing.